pkg/cmd/server/server.go

```
@@ -171,7 +171,7 @@ func (c *Config) Complete(ctx context.Context) (RunnableServer, error) {
 			log.Trace().Int(fmt.Sprintf("preshared-key-%d-length", index+1), len(presharedKey)).Msg("preshared key configured")
 		}
 
-		c.GRPCAuthFunc = auth.RequirePresharedKey(c.PresharedKey)
+		c.GRPCAuthFunc = auth.MustRequirePresharedKey(c.PresharedKey)
```

same here: `Complete` has an error in the signature, so let's return an error